ios/app.xcodeproj/project.pbxproj

```diff
@@ -33,7 +34,6 @@
 		14310463206844D0002E158B /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 14310423206844D0002E158B /* MaterialIcons.ttf */; };
 		1431046A206845BF002E158B /* libRNVectorIcons.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 14310469206845BA002E158B /* libRNVectorIcons.a */; };
 		146834051AC3E58100842450 /* libReact.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 146834041AC3E56700842450 /* libReact.a */; };
-		148B64C82086D86A0005BA9A /* libRCTMaterialKit.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 148B64C72086D5A10005BA9A /* libRCTMaterialKit.a */; };
```

Comment:
I don't remember, did we figure out if these changes are necessary or if they should be committed?